### DISCUSSION TITLE
general/http-cache

### DISCUSSION THREAD

**Discussion by giscus** [howdy-mj/blog-v2#48] [Announcements]
# general/http-cache

HTTP 캐시 훑어보기

https://www.howdy-mj.me/general/http-cache

**Comment by Jonghakseo** [comment]
사실 캐시를 적용하는건 그리 어렵지 않은데, 적절한 시기에 리소스별 Cache-Busting을 일으켜서 최신화된 정보를 제공하는 부분은 보다 생각할 거리가 있는것 같습니다. 잘 읽었습니다~!
